.github/workflows/build_scheduled.yml

```diff
@@ -0,0 +1,371 @@
+        run: packer init aws.pkr.hcl
+
+      - name: Build the AWS AMI using Packer (${{ matrix.arch }})
+        run: packer build aws.pkr.hcl
+        env:
+            PKR_VAR_encrypt_boot: false
+            PKR_VAR_ami_name_prefix: spacelift-${{ needs.timestamp.outputs.timestamp }}
+            PKR_VAR_source_ami_architecture: ${{ matrix.arch }}
+            PKR_VAR_instance_type: ${{ matrix.arch == 'x86_64' && 't3.micro' || 't4g.micro' }}
+
+      - name: Upload manifest
+        uses: actions/upload-artifact@v4
+        with:
+          path: manifest_aws_${{ matrix.arch }}.json
+          name: manifest_aws_${{ matrix.arch }}.json
+          retention-days: 5
+
+  aws-govcloud:
+    # Since we run in parallel, let's make sure we use the same timestamp for all jobs
+    needs: timestamp
+    strategy:
+      matrix:
+        arch: [x86_64, arm64]
+    name: Build the AWS (GovCloud) AMI using Packer
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Check out the source code
+        uses: actions/checkout@main
+
+      - name: Configure GovCloud AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: ${{ secrets.GOVCLOUD_AWS_REGION }}
+          role-to-assume: ${{ secrets.GOVCLOUD_AWS_ROLE_ARN }}
+          role-duration-seconds: 3600
+
+      - name: Setup packer
+        uses: hashicorp/setup-packer@main
+        with:
+          version: latest
+
+      - name: Initialize Packer
+        run: packer init aws.pkr.hcl
+
+      - name: Build the GovCloud AWS AMI using Packer (${{ matrix.arch }})
+        run: packer build aws.pkr.hcl
+        env:
+          PKR_VAR_source_ami_owners: '["045324592363"]'
+          PKR_VAR_region: us-gov-east-1
+          PKR_VAR_ami_regions: '["us-gov-east-1", "us-gov-west-1"]'
+          PKR_VAR_encrypt_boot: false
+          PKR_VAR_ami_name_prefix: spacelift-${{ needs.timestamp.outputs.timestamp }}
+          PKR_VAR_source_ami_architecture: ${{ matrix.arch }}
+          PKR_VAR_instance_type: ${{ matrix.arch == 'x86_64' && 't3.micro' || 't4g.micro' }}
+
+  azure:
+    name: Build Azure AMI using Packer
+    runs-on: ubuntu-latest
+    env:
+      PKR_VAR_client_id: "976e4a6e-c619-417e-9add-50e2d674e2db"
+      PKR_VAR_tenant_id: ${{ secrets.AZURE_TENANT_ID }}
+      PKR_VAR_subscription_id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+      PKR_VAR_image_resource_group: rg-worker_images-public-westeurope
+      PKR_VAR_packer_work_group: rg-worker_images_packer-public-westeurope
+      PKR_VAR_gallery_resource_group: rg-worker_images-public-westeurope
+      PKR_VAR_gallery_name: spacelift_worker_images_public
+      PKR_VAR_gallery_image_name: spacelift_worker_image
+      PKR_VAR_gallery_replication_regions: '["westeurope", "northeurope", "northcentralus", "eastus", "eastus2", "westus2", "westus3"]'
+      PKR_VAR_gallery_image_version: 1.0.${{ github.run_number }}
```

Comment:
one thing to keep in mind that we'll have a new versioning here since this is a brand new job. this is fine because we're publishing into a brand new gallery so the versioning can be different here.

Reply:
we _could_ bump the version to `v2.0.x` 🤷

Reply:
Let's do the major version bump. I think it'll be less confusing than having overlapping version numbers.

Reply:
bumped it ✔️